### DISCUSSION TITLE
Why trying to connect to restricted VC doesn't raise an error?

### DISCUSSION THREAD

**Discussion by solaluset** [Pycord-Development/pycord#2129] [Q&A] [answered]
When the bot tries to connect to restricted VC, it doesn't throw an error and just hangs. Afaik, any operation performed without permissions throws a permission error, why this is exceptional?

```python
@bot.command()
async def connect(ctx):
    await ctx.author.voice.channel.connect()
    await ctx.respond("Connected")
```

I've already asked this on Discord, but didn't get the answer.
The docs actually don't mention permission errors in `.connect()`, is this intended behaviour?

**Comment by Lulalaby** [comment, marked as answer]
If you asked on our discord, then wait for an answer as per our help rules.
